fdbserver/masterserver.actor.cpp

```diff
@@ -965,6 +968,7 @@ ACTOR Future<Void> sendInitialCommitToResolvers(Reference<MasterData> self) {
 	}
 
 	wait(waitForAll(replies));
+	TraceEvent("SentTxnStateStoreToResolvers", self->dbgid);
```

Comment:
Master doesn't send txnStateStore to resolvers.
```suggestion
	TraceEvent("InitializedAllResolvers", self->dbgid);
```

Reply:
Yes. It sends LastEpochEnd version to resolvers.